version-control-login.rst

```diff
@@ -0,0 +1,43 @@
+.. post:: May 9, 2018
+   :tags: feature, new-user, versioncontrol
+   :author: David
+   :location: SAN
+
+
+Version Control Log in
+======================
+
+Today we are announcing the ability to log in or sign up to Read the Docs
+with your favorite version control hosting services like GitHub, BitBucket, or GitLab.
+This was one of our most requested features and
+it has been something we've been meaning to launch for a long time.
+
+
+For new Read the Docs users
+---------------------------
+
+For new users, the sign up process is significantly streamlined.
+There's no new password to remember and when you're ready to start building your docs,
+Read the Docs will be ready with a list of your repositories to get started.
+
+Click the "sign up" button on the Read the Docs home page and then click
+the button for your favorite version control host.
```

Comment:
I think it's more accurate to use the same wording you used in the first paragraph:

"version control hosting service."